modules/luci-base/po/cs/base.po

```diff
@@ -1,5 +1,5 @@
 msgid ""
-msgstr ""
+
```

Comment:
This empty line looks erroneous. I think that it should have the ` msgstr "" ` item like earlier.

And please sign-off-by your commit properly.